geniza/corpus/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class DocsConfig(AppConfig):
+    name = 'corpus'
```

Comment:
Might be missing something, but why isn't this called `CorpusConfig`?

Reply:
Ha, good catch, thank you. (It's because I created it before I got the corpus name from Marina and I missed it when I renamed everything)